.gitignore

```diff
@@ -58,7 +58,6 @@ typings/
 
 demo/testConfig.js
 
-.idea
```

Comment:
Should we commit this? In case of vsc launch.json supposed to be edited by user but in Webstorm .idea folder is hidden and it's content maybe depends from OS

Reply:
> Should we commit this? In case of vsc launch.json supposed to be edited by user but in Webstorm .idea folder is hidden and it's content maybe depends from OS

@andriichumak  what do you think?

Reply:
We should follow JetBrains guide as for which files inside .idea must be committed to Git. See:
https://intellij-support.jetbrains.com/hc/en-us/articles/206544839-How-to-manage-projects-under-Version-Control-Systems
https://github.com/github/gitignore/blob/master/Global/JetBrains.gitignore